### DISCUSSION TITLE
Inventory discrepancy's

### DISCUSSION THREAD

**Discussion by shanemeisnerdhg** [microsoft/powerapps-tools#1129] [Q&A]
After setting up the CoE Kit, all flows ran without errors, but when looking at the Apps in the CoE dashboard ,it wasn't showing all the Apps that I could see in make.powerapps.com. The Help document mentioned "FullInventory":

**FullInventory -Determines whether you want to update only objects that have changed, or all objects. 
Switching to Yes will cause the flows to inventory every single app, flow, and bot in the tenant every day, and isn't recommended for large tenants.**

So I changed that to "Yes" on Friday.. 
I checked this morning and all flows show to have succeeded. 
When I go to make.powerapps.com I see a couple of apps that were created over a year ago.
![image](https://user-images.githubusercontent.com/97987769/152801189-43f16c72-83b1-4ad0-8b3d-03bd4533b8eb.png)

But, when I go to the CoE dashboard, those two apps highlighted above shot do not show in the dashboard:
![image](https://user-images.githubusercontent.com/97987769/152800945-142d793a-b9cf-41ba-845c-d99b5ba63996.png)

Any thoughts??

**Comment by manuelap-msft** [comment]
Hello, this repo is no longer used for the CoE Starter Kit. Please raise your issue or question in our dedicated repo here: https://github.com/microsoft/coe-starter-kit/issues
